packages/features/webhooks/components/EventTypeWebhookListItem.tsx

```diff
@@ -0,0 +1,97 @@
+import classNames from "@calcom/lib/classNames";
+import { useLocale } from "@calcom/lib/hooks/useLocale";
+import type { WebhookTriggerEvents } from "@calcom/prisma/enums";
+import { Badge, Button, Switch, Tooltip } from "@calcom/ui";
+
+type WebhookProps = {
+  id: string;
+  subscriberUrl: string;
+  payloadTemplate: string | null;
+  active: boolean;
+  eventTriggers: WebhookTriggerEvents[];
+  secret: string | null;
+  eventTypeId: number | null;
+  teamId: number | null;
+};
+
+export default function EventTypeWebhookListItem({
+  webhook,
+  index,
+  canEditWebhook = true,
+  lastItem,
+  onEditWebhook,
+  onToggleWebhook,
+  onDeleteWebhook,
+}: {
+  webhook: WebhookProps;
+  index: number;
+  canEditWebhook?: boolean;
+  lastItem: boolean;
+  onEditWebhook: (webhook: WebhookProps) => void;
+  onToggleWebhook: (updatedWebhook: WebhookProps) => void;
+  onDeleteWebhook: (updatedWebhook: WebhookProps) => void;
+}) {
+  const { t } = useLocale();
+
+  return (
+    <div className={classNames("flex w-full justify-between p-4", lastItem ? "" : "border-subtle border-b")}>
+      <div className="w-full truncate">
+        <div className="flex">
+          <Tooltip content={webhook.subscriberUrl}>
+            <p className="text-emphasis max-w-[600px] truncate text-sm font-medium">
+              {webhook.subscriberUrl}
+            </p>
+          </Tooltip>
+          <Badge variant="gray" className="ml-2">
+            {t("readonly")}
```

Comment:
Why always show the readonly badge here?

Reply:
Yeah that's something that slipped me. Will handle